reports/m4_reflection.md

```diff
@@ -0,0 +1,52 @@
+## Milestone 4: Reflection
+
+So far, we have consistently adhered to the best practices for visualization we learned in 532. We have successfully implemented most of the components outlined in our initial sketch. The only feature yet to be implemented is the CO2 emissions per capita chart. The decision to delay this implementation isn't due to technical challenges; rather, we intend to reserve it as a key enhancement for future versions of the dashboard. The current version remains functional and intuitive, ready to be launched for potential users.
+
+For Milestone 4, we have accomplished the following improvements:
+
+### Major changes
+
+- Add an additional output plot of co2 emissions per capita
+
+- Remove Antarctica from map to improve scaling and allow map to occupy more horizontal white space.
+
+- Changed our processed data file to parquet format to improve dashboard response time
+
+- Add the selected year range to the output text
+
+### Minor changes
+
+- Add a note explaining the average temperature change metric
+
+- Set default tool as box select in world map
+
+
```

Comment:
```suggestion
- Change some spacing in the UI

- Fixed the map to not allow panning

- Change so that units are consistent (GigaTons)

- Add selected year as description for clarity
```